testing/services/dummy/dsubscriber/clients/agent.go

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
Wouldn't it make more sense to move ukama agent CDR client under common/rest/client so that it could be use by any other service that might need i?

Reply:
It's node-gateway integration. I'm sure if any other system would want to push cdr's to node-gateway.